src/main/java/com/project/reha/service/PatientService.java

```diff
@@ -1,13 +1,26 @@
 package com.project.reha.service;
 
 import com.project.reha.dao.PatientDao;
-import com.project.reha.dao.ProcedureDao;
+import com.project.reha.model.Patient;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.util.Optional;
+
 @Service
 @RequiredArgsConstructor
 public class PatientService {
     private final PatientDao patientDao;
 
+    public void createPatient(Patient patient) {
```

Comment:
On the Service layer you should use another type of objects (not persistence). You should create another objects layer with dto( data transient objects) and your service will return for example PatientDto but DAO will return PatientPO. This PatientDto will be used for UI too.